packages/app/src/cli/services/bulk-operations/run-query.ts

```diff
@@ -19,6 +19,7 @@ export async function runBulkOperationQuery(
     query: BulkOperationRunQuery,
     session: adminSession,
     variables: {query},
+    version: '2026-01',
```

Comment:
(Need to use `2026-01` or later to get access to `bulkOperation`. Maybe later this will be a parameter, but for now hardcoding `2026-01` is a simple way to make sure things work against the right version.)